server/server/Controllers/adminCrud.js

```diff
@@ -0,0 +1,85 @@
+const itemsModel  = require("../models/Item");
+
+// Admin get item by id
+const getItemById = async (req, res) => {
+    const { id } = req.params;
+    try {
+        const item = await itemsModel.findById(id);
+        if (!item) {
+            // If the item is not found
+            return res.status(404).json({ message: "Item not found" });
+        }
+        // If the item is found
+        res.json(item);
+    } catch (err) {
+        // If there's an error
+        res.status(500).json({ message: err.message });
+    }
+};
+
+
+
+// admin add new item
+const addNewItem = async (req, res) => {
+    const ItemData = req.body;
+    try {
+      const newItem = await itemsModel.create(ItemData);
```

Comment:
To avoid unintentional errors, I think that any required data that is not provided or any unusual data passed to the `create` function should be processed first from the request body.

Reply:
done